src/app/components/user/forgotten-password/forgotten-password.component.scss

```diff
@@ -49,3 +49,19 @@
     cursor: pointer;
   }
 }
+
+
+@media (max-width: 430px) {
+    .container{
```

Comment:
put spaces everywhere before the opening bracket "{"

Reply:
done